### ISSUE TITLE
ListStatus not accepting envelope_ids

### ISSUE THREAD

**Issue by tedPen** [docusign/docusign-ruby-client#15]
This appears to be covered by this issue in the related c# client: [issue 196](https://github.com/docusign/docusign-csharp-client/issues/196). I want to confirm that if an update gets pushed out for that codebase it will be pulled into the ruby client.

To reiterate the issue I have code that is trying to make a call to list_status to only pull in envelope_ids that I care about hover I receive an error message indicating that a date range is required or envelope ids or transaction ids need to be specified. Here is my code in context:
```ruby
# api_client is an authenticated instance of DocuSign_eSign::ApiClient
envelope_api = DocuSign_eSign::EnvelopesApi.new(api_client)
request = DocuSign_eSign::EnvelopeIdsRequest.new
request.envelope_ids = envelope_ids # envelope_ids is an array of envelope id strings
response = envelope_api.list_status(login_account.account_id, request)
```

I'm working around the issue by issuing a request with the from_date specified by I want to get status information about specific envelopes to limit the number of calls I need to make. 

As a side note it would be great if you could configure your code generate to use more standard ruby conventions, for example, the namespace DocuSign_eSign is a strange combination of camelcase and underscores.

**Comment by tedPen** [comment]
@RajRele any update on a timeline here?

**Comment by LarryKlugerDS** [comment]
Unfortunately we don't have a solid schedule for the Ruby SDK release that will fix this problem.

The workaround is to call the API directly. You can use either the [Envelopes::listStatusChanges](https://developers.docusign.com/esign-rest-api/reference/Envelopes/Envelopes/listStatusChanges) or the [Envelopes::listStatus](https://developers.docusign.com/esign-rest-api/reference/Envelopes/Envelopes/listStatus) api methods.

For the latter, the list of envelope_ids is supplied as the request body, and the query parameter `envelope_ids` must be set to `request_body`

**Comment by tedPen** [comment]
Is there any update on this issue? Would be great to have access to this through the client.

**Comment by LarryKlugerDS** [comment]
Sorry for the slow reply. Both issues are still in plan, but awaiting resources. Unfortunately I do not have a planned schedule date at this time.

**Comment by LarryKlugerDS** [comment]
Good news, this issue has been fixed with the 3.0.0.rc.1 release of the SDK and [gem](https://rubygems.org/gems/docusign_esign/versions/3.0.0.rc1). The source is available as branch [ngopala/v2.1-19.2.02](https://github.com/docusign/docusign-ruby-client/tree/ngopala/v2.1-19.2.02). 

It will be released to master and the gem will be updated in the next week.

Please comment on this issue if you spot any issues with the RC (release candidate) or the forthcoming release.

Thanks,
Larry
